### PR TITLE
docs: scripts and kafkacat quickstarts updates

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

This PR updates the _Kafka scripts_ and _Kafkacat_ quick starts. 

- Changes the titles to _Configuring and connecting..._ rather than _Using..._ 
- Replaces _bin scripts_ with _scripts_  
- Removes the download/install step, which becomes a general prereq. Other detail moved to other sections of each quick start
- Adds a line about using the Kafka scripts to check a new Kafka instance to _Getting Started_
- Removes group Id config (not currently working)

NOTE
- The PR doesn't change Kafkacat to kcat (pending a decision on that)
- The version check (`-version` and `-V`) is moved to a portal-only section

Details: [MGDSTRM-5099](https://issues.redhat.com/browse/MGDSTRM-5099)

PORTAL DOC
- [Kafkacat quick start](http://file.emea.redhat.com/pmellor/docs/managed-kafka-drafts/kafkacat-quickstart.html)
- [Kafka scripts quick start](http://file.emea.redhat.com/pmellor/docs/managed-kafka-drafts/kafka-scripts-quickstart.html)

### CODE REVIEW

File: kafka-bin-scripts/README.adoc [hunk 1]
```diff
@@ -143,23 +116,22 @@ security.protocol=SASL_SSL
...
 [id="proc-producing-messages-kafka-bin-scripts_{context}"]
-== Producing messages using Kafka bin scripts
+== Producing messages using Kafka scripts
 
 You can use the `kafka-console-producer` script to produce messages to Kafka topics.
 
 .Prerequisites
 
-* The Kafka bin scripts are downloaded.
+* You have checked that the `kafka-console-producer` and `kafka-topics` scripts are in the `bin/` directory of your Kafka download.
```
Comment (PaulRMellor) [inline kafka-bin-scripts/README.adoc:134]: Not sure we need this. There was something similar in the download step.

File: kafka-bin-scripts/README.adoc [hunk 2]
```diff
@@ -217,13 +189,14 @@ ifndef::qs[]
...
 [id="proc-consuming-messages-kafka-bin-scripts_{context}"]
-== Consuming messages using Kafka bin scripts
+== Consuming messages using Kafka scripts
 
 You can use the `kafka-console-consumer` script to consume messages from Kafka topics. This example consumes the messages that you sent previously with the producer that you created with the `kafka-console-producer` script.
 
 .Prerequisites
 
 * You used the `kafka-console-producer` script to produce example messages to a topic.
+* You have checked that the `kafka-console-consumer` script is in the `bin/` directory of your Kafka download.
```
Comment (PaulRMellor) [inline kafka-bin-scripts/README.adoc:199]: Not sure we need this. There was something similar in the download step.

File: kafka-bin-scripts/README.adoc (hunk 1 above)
Comment (wtrocki) [reply kafka-bin-scripts/README.adoc:134]: Why users will have kafka download on the machine? typically they do not run kafka - getting managed version so we might need to rephrase it to actually download kafka binary (telling them where to do it)

**Comment by wtrocki** [comment]
> Removes the download/install step, which becomes a general prereq. Other detail moved to other sections of each quick start

I'm not sure about this one? Is there any change outside this that I'm missing (like previous step that they will get that binary)

File: getting-started/README.adoc [hunk 3]
```diff
@@ -212,6 +212,8 @@ endif::[]
 ifdef::qs[]
 [#conclusion]
 Congratulations! You successfully completed the {product} Getting Started quick start, and are now ready to use the service.
+
+You can use Kafka scripts to check that you can connect with your Kafka instance.
```
Comment (PaulRMellor) [inline getting-started/README.adoc:216]: @jschuler Wondering if we can have 2 x _next quick start_ links from this quick start. We could then suggest Kafkacat here too.

File: kafka-bin-scripts/README.adoc (hunk 1 above)
Comment (wtrocki) [reply kafka-bin-scripts/README.adoc:134]: To answer - once we put proper URL to download we do not need that step at all.

File: kafka-bin-scripts/README.adoc (hunk 2 above)
Comment (wtrocki) [reply kafka-bin-scripts/README.adoc:199]: No need

**Comment by wtrocki** [comment]
Rebase needed.

File: kafka-bin-scripts/README.adoc [hunk 4]
```diff
@@ -46,74 +46,49 @@ ifdef::context[:parent-context: {context}]
...
 
-. Check the version number of the `kafka-console-producer` script to verify that the scripts were downloaded correctly.
-+
-.Verifying Kafka bin scripts
+.Verifying Kafka scripts
 [source]
 ----
 $ ./kafka-console-producer.sh --version
 2.7.0 (Commit:448719dc99a19793)
 ----
```
Comment (sterobin) [inline kafka-bin-scripts/README.adoc:78]: @PaulRMellor I'm wondering if we could streamline the prereq a bit, something like this. I know it's not too common to put in a snippet in a prereq, but I've done it a couple of times and have enjoyed the payoff it brings. But up to you of course.

```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
* You have a running Kafka instance in {product}.
* You have downloaded the latest binary version of the https://kafka.apache.org/downloads[Apache Kafka distribution^] and verified the installation.
+
.Verifying Kafka scripts
[source]
----
$ ./kafka-console-producer.sh --version
2.7.0 (Commit:448719dc99a19793)
----
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
```

Or possibly this:


```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
* You have a running Kafka instance in {product}.
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
* You have downloaded the latest binary version of the https://kafka.apache.org/downloads[Apache Kafka distribution^] and verified the installation.
+
.Verifying Kafka scripts
[source]
----
$ ./kafka-console-producer.sh --version
2.7.0 (Commit:448719dc99a19793)
----
```

I'd suggest the same approach for the prereq in the Kafkacat bit too.

File: kafkacat/README.adoc [hunk 5]
```diff
@@ -91,9 +80,14 @@ Copyright (c) 2014-2019, Magnus Edenhill
 Version 1.6.0 (JSON, Avro, Transactions, librdkafka 1.6.1 builtin.features=gzip,snappy,ssl,sasl,regex,lz4,sasl_gssapi,sasl_plain,sasl_scram,plugins,zstd,sasl_oauthbearer)
 ----
```
Comment (sterobin) [inline kafkacat/README.adoc:81]: @PaulRMellor Same suggestion as before with scripts, maybe streamline the prereq as follows:

```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
//* You have a subscription to {product-long}. For more information about signing up, see *<@SME: Where to link?>*.
* You have a running Kafka instance in {product}.
* You have installed the latest version of https://github.com/edenhill/kafkacat[Kafkacat^] for your operating system and verified the installation.
+
.Verifying Kafkacat installation
[source]
----
$ kafkacat -V

kafkacat - Apache Kafka producer and consumer tool
https://github.com/edenhill/kafkacat
Copyright (c) 2014-2019, Magnus Edenhill
Version 1.6.0 (JSON, Avro, Transactions, librdkafka 1.6.1 builtin.features=gzip,snappy,ssl,sasl,regex,lz4,sasl_gssapi,sasl_plain,sasl_scram,plugins,zstd,sasl_oauthbearer)
----
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
```

Or this:

```
.Prerequisites
ifndef::community[]
* You have a Red Hat account.
endif::[]
//* You have a subscription to {product-long}. For more information about signing up, see *<@SME: Where to link?>*.
* You have a running Kafka instance in {product}.
* https://adoptopenjdk.net/[JDK^] 11 or later is installed.
* For Windows, the latest version of https://www.oracle.com/java/technologies/javase-downloads.html[Oracle JDK^] is installed.
* You have installed the latest version of https://github.com/edenhill/kafkacat[Kafkacat^] for your operating system and verified the installation.
+
.Verifying Kafkacat installation
[source]
----
$ kafkacat -V

kafkacat - Apache Kafka producer and consumer tool
https://github.com/edenhill/kafkacat
Copyright (c) 2014-2019, Magnus Edenhill
Version 1.6.0 (JSON, Avro, Transactions, librdkafka 1.6.1 builtin.features=gzip,snappy,ssl,sasl,regex,lz4,sasl_gssapi,sasl_plain,sasl_scram,plugins,zstd,sasl_oauthbearer)
----
```

**Review by sterobin — COMMENTED** [review]
@PaulRMellor Looks great to me. Just a couple minor comments. Also, with this change, do you want to raise a PR to update the [feature file](https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/blob/quickstarts/src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature) as well accordingly? To reflect the terminology and prereq changes etc for parity.

File: kafka-bin-scripts/README.adoc [hunk 6]
```diff
@@ -46,74 +46,49 @@ ifdef::context[:parent-context: {context}]
...
+With the scripts, you can produce and consume messages, and perform various operations against the Kafka APIs to administer topics, consumer groups, and other resources.
+
+NOTE: The command examples in this quick start demonstrate how to use the Kafka scripts on Linux and macOS. If you're using Windows, use the Windows versions of the scripts. For example, instead of the `__<Kafka-distribution-dir>__/bin/kafka-console-producer.sh` script, use the `__<Kafka-distribution-dir>__\bin\windows\kafka-console-producer.bat` script.
 
 .Prerequisites
 ifndef::community[]
 * You have a Red Hat account.
 endif::[]
 * You have a running Kafka instance in {product}.
+* You have downloaded the latest binary version of the https://kafka.apache.org/downloads[Apache Kafka distribution^].
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply kafka-bin-scripts/README.adoc:67]: Thanks @wtrocki  and @sterobin - I'm for avoiding a specific version if possible. Would _latest supported binary version_ make things better? Or maybe that justs raise the question, which?

[the remark this answers is not in the document]
Comment (PaulRMellor) [reply kafka-bin-scripts/README.adoc:67]: @wtrocki - Hi Wojciech. I raised this in our MK CCS meeting.  Result: we're looking to get versions into a _supported configurations_ article and keep them out of the doc.

File: kafka-bin-scripts/README.adoc (hunk 4 above)
Comment (PaulRMellor) [reply kafka-bin-scripts/README.adoc:78]: I did think about that and I've used them before myself. Wasn't sure of the _quick start_ convention, so happy to go with option B here. :+1:

File: kafkacat/README.adoc (hunk 5 above)
Comment (PaulRMellor) [reply kafkacat/README.adoc:81]: Done

File: kafka-bin-scripts/README.adoc (hunk 1 above)
Comment (PaulRMellor) [reply kafka-bin-scripts/README.adoc:134]: Removed

File: kafka-bin-scripts/README.adoc (hunk 2 above)
Comment (PaulRMellor) [reply kafka-bin-scripts/README.adoc:199]: Removed

**Comment by wtrocki** [comment]
Feel free to ignore my comments. I will lean on your expertise.

**Comment by PaulRMellor** [comment]
> Feel free to ignore my comments. I will lean on your expertise.
> 
> EDIT:
> Closed by mistake :D

Cheers @wtrocki . Changes pushed in now.

**Review by sterobin — APPROVED** [review]
@PaulRMellor Good with me. And I see your other PR to update the feature file accordingly. Thanks!

**Review by wtrocki — APPROVED** [review]
